crypto/dh/dh_rfc7919.c

```diff
@@ -22,6 +22,7 @@ static DH *dh_param_init(const BIGNUM *p, int32_t nbits)
     dh->p = (BIGNUM *)p;
     dh->g = (BIGNUM *)&_bignum_const_2;
     dh->length = nbits;
+    dh->dirty_cnt++;
```

Comment:
Welllll...  it does need to increment from zero.

Reply:
Why?

Reply:
The initial EVP_PKEY dirty_cnt copy will be zero.  I would rather an initial DH actually look dirty, i.e. have a different dirty_cnt, to avoid any uncertainty that the contents of the DH should effectively be exported.